zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/GroupController.java

```diff
@@ -153,6 +154,9 @@ public CompletableFuture<ResponseEntity<Object>> updateGroup(
         () ->
             groupServices
                 .withAuthentication(RequestMapper.getAuthentication())
-                .updateGroup(updateGroupRequest.groupKey(), updateGroupRequest.name()));
+                .updateGroup(
+                    updateGroupRequest.groupId(),
+                    updateGroupRequest.name(),
+                    updateGroupRequest.description()));
```

Comment:
🔧 I'd just pass the DTO along instead of all fields of the DTO separately.

Reply:
I think I can use the same DTO for creation and update. I'll create a followup issue to refactor both 👍